build.bat

```diff
@@ -21,23 +21,38 @@ rem Build with nuget, it solves the directory structure for us.
 call .\Tools\nuget\build.bat %ARCH_OPT% %PGO_OPT% %REBUILD_OPT%
 
 rem Install with nuget into a build folder
-if "%ARCH_OPT%" EQU "-x64" (.\externals\windows-installer\nuget\nuget.exe install python -Source %~dp0\PCbuild\amd64 -OutputDirectory nuget-result) 
-else (.\externals\windows-installer\nuget\nuget.exe install pythonx86 -Source %~dp0\PCbuild\win32 -OutputDirectory nuget-result)
+set NUGET_PYTHON_PACKAGE_NAME=python
+set ARCH_NAME=amd64
+if "%ARCH_OPT%" EQU "-x86" (
+    set NUGET_PYTHON_PACKAGE_NAME=pythonx86
+    set ARCH_NAME=win32
+)
+
+.\externals\windows-installer\nuget\nuget.exe install %NUGET_PYTHON_PACKAGE_NAME% -Source %~dp0\PCbuild\%ARCH_NAME% -OutputDirectory nuget-result-%NUGET_PYTHON_PACKAGE_NAME%
+
+@REM if "%ARCH_OPT%" EQU "-x64" (
```

Comment:
Please remove.

Reply:
OK, of course.  I removed commented block, also makes 
Win32 and x64 versions builded in parallel ( `output` and `output32` folders)